---a/net/minecraft/world/ILockableContainer.java
+++b/net/minecraft/world/ILockableContainer.java
@@ -2,12 +2,12 @@
 
 import net.minecraft.inventory.IInventory;
 
-public interface ILockableContainer extends IInventory, IInteractionObject
-{
+public interface ILockableContainer extends IInventory, IInteractionObject {
+
     boolean isLocked();
 
     void setLockCode(LockCode code);
 
     LockCode getLockCode();
-}
+}
